src/pages/RegisterPage.js

```diff
@@ -1,21 +1,42 @@
 import React from 'react';
 import { Grid, Row, Col } from 'react-flexbox-grid';
+import { reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getTranslator } from '../reducers/translation';
 import HeaderRegistration from '../components/HeaderRegistration';
+import { SquareButton } from '../components/Buttons';
+import TextField from '../components/FieldText';
 import Footer from '../components/Footer';
-
 import common from '../styles/common.scss';
+import RegisterPageStyle from './RegisterPage.scss';
+import { LoginLink } from '../components/PredefinedLinks';
 
-export default () =>
+export const RegisterFormView = ({ handleSubmit, i18n }) =>
   (<div className={common.layoutAppContainer}>
     <HeaderRegistration />
     <div className={`${common.layoutContentArea} ${common.gradientContentArea}`}>
       <div className={common.layoutWidthLimiter}>
-
         <Grid fluid>
           <Row>
             <Col lgOffset={3} lg={6}>
               <div className={common.whiteArea}>
-                content
+                <div className={RegisterPageStyle.columnArea}>
+                  <h2>{i18n('Create your private account first')}</h2>
```

Comment:
It's better to have the heading inside the form [documentation](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/How_to_structure_an_HTML_form).
I we can also consider applying the `RegisterPageStyle.columnArea` style directly to `form`, that way we can eliminate a `div`.